plugins/generator-1.19.4/forge-1.19.4/templates/projectile/projectile.java.ftl

```diff
@@ -166,21 +162,18 @@ public class ${name}Entity extends AbstractArrow implements ItemSupplier {
 		double dx = target.getX() - entity.getX();
 		double dy = target.getY() + target.getEyeHeight() - 1.1;
 		double dz = target.getZ() - entity.getZ();
-		entityarrow.shoot(dx, dy - entityarrow.getY() + Math.hypot(dx, dz) * 0.2F, dz, ${data.bulletPower}f * 2, 12.0F);
+		entityarrow.shoot(dx, dy - entityarrow.getY() + Math.hypot(dx, dz) * 0.2F, dz, ${data.power}f * 2, 12.0F);
 
 		entityarrow.setSilent(true);
-		entityarrow.setBaseDamage(${data.bulletDamage});
-		entityarrow.setKnockback(${data.bulletKnockback});
-		entityarrow.setCritArrow(${data.bulletParticles});
-		<#if data.bulletIgnitesFire>
+		entityarrow.setBaseDamage(${data.damage});
+		entityarrow.setKnockback(${data.knockback});
+		entityarrow.setCritArrow(${data.showParticles});
+		<#if data.igniteFire>
 			entityarrow.setSecondsOnFire(100);
 		</#if>
 		entity.level.addFreshEntity(entityarrow);
-
-		<#if data.actionSound.toString()?has_content>

```

Comment:
Before, action sound was optional, and this should stay so (no call to this if not needed)